imaging/ml/ml_codelab/breast_density_auto_ml.ipynb

```diff
@@ -260,16 +128,6 @@
     "gsutil -q iam ch serviceAccount:${SERVICE_ACCOUNT}:objectCreator $3"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "colab_type": "text",
-    "id": "aIfKnxqHjE9A"
-   },
-   "source": [
-    "Next we will convert the DICOMs to JPEGs using the ExportDicomData API. This is an asynchronous call that returns an Operation name. You can use the Operation name to poll the status of the operation."
```

Comment:
Not sure why the other codelab has this. If you keep this for both. Please add a link. https://cloud.google.com/sdk/gcloud/reference/beta/healthcare/dicom-stores/export/gcs

Reply:
done